src/client/Python/msrest/readme.rst

```diff
@@ -15,6 +15,13 @@ To install:
 Release History
 ---------------
 
+2016-09-01 Version 0.4.3
+++++++++++++++++++++++++
+
+**Bugfixes**
+
+- Better exception message (https://github.com/Azure/autorest/pull/1300)
```

Comment:
should we point to the [issue](https://github.com/Azure/azure-sdk-for-python/issues/592), rather this pull request, which is still active?